### PR TITLE
Azure docs update

### PR DESCRIPTION
**Pull request by LBegnaud**
Started reading through the Azure deployment docs to update our process to not have to upload a vhd each time we create a vyos VM. Found some things we do that might be useful to have in the docs.

### CODE REVIEW

**Review by zdc — APPROVED** [review]
Everything is OK with this change.
@LBegnaud, you also may want to extend docs about Azure with a bit more detailed information about how to use routing tables with VyOS in Azure properly. I believe this is one of the parts that need to be explained well.

**Comment by LBegnaud** [comment]
I have a section the about how to make a route table and apply it, I guess you're saying to add screenshots and expand on the explanation? I didn't want to get too wordy with it

**Comment by zdc** [comment]
I mean something like this articles series: https://aws.amazon.com/blogs/apn/amazon-vpc-for-on-premises-network-engineers-part-one/, but in one-two paragraphs and only what is important for VyOS deployment.
The biggest problem for customers who do not know how networks work in clouds is to understand that there is no plain L2 network between devices and even traffic inside the same "L2" segment obey policies configured in a cloud console. The same about routing, for example - it is not possible to have different routing behavior for devices in the same subnet because as soon as traffic leaves the virtual appliance's interface, it will be routed according to the routing table for a subnet, no matter of what was configured locally. And so on.
